.github/workflows/ci.yml

```diff
@@ -306,6 +309,9 @@ jobs:
           sudo mkdir .aws && sudo chmod 777 .aws && sudo chown 1000 .aws
           popd
           ./hadoop-ozone/dev-support/checks/kubernetes.sh
+      - name: Summary of failures
+        run: cat target/${{ github.job }}/summary.txt
+        if: ${{ !cancelled() }}
```

Comment:
I have a quick question, why is it that for integration test we use
```
      - name: Summary of failures
        run: cat target/${{ github.job }}/summary.txt
        if: always()
```
Shouldn't that also be only if the job is not canceled?

Reply:
Right, it should be.

Reply:
Thanks @aswinshakil for pointing this out.  Patch is updated, can you please take another look?